examples/react-native/package.json

```diff
@@ -19,7 +19,7 @@
     "@aws-amplify/ui-react-native": "*",
     "@react-native-async-storage/async-storage": "^1.17.5",
     "@react-native-community/netinfo": "^8.3.0",
-    "@xstate/react": "3.0.0",
+    "@xstate/react": "3.0.1",
```

Comment:
Why do we need `xstate/react` in this package (or _@aws-amplify/ui-react_)? Shouldn't the dep in _@aws-amplify/ui-react-core_ cover it?

Reply:
+1

Reply:
Yeah, true.
We don't need to have it again.
Updated PR.